src/components/dataPanel/PanelViewingState.tsx

```diff
@@ -18,25 +19,23 @@ export interface IField {
   value: IFieldValue | IFieldValue[];
 }
 
-interface IPanelViewStateProps {
+export interface IPanelViewStateProps {
   fields: IField[];
   columns?: 1 | 2;
   splitAfter?: number;
 }
 
-export enum DataPanelStates {
-  Loading = "LOADING",
-  Viewing = "VIEWING",
-  Editing = "EDITING",
-  Error = "ERROR",
-}
```

Comment:
Moved to own `States.ts` file in `dataPanel/` as a higher level concern.  I didn't feel states about the _whole_ data panel belonged in the file that defined just the _viewing_ state.